boards/BMS/Test/Inc/imd_fff.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
Transferring unresolved comment from https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/536#discussion_r405192994

Reply:
Seems reasonable to me, I'd say go for it.